frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard-steps/select-capacity-and-nodes-step.tsx

```diff
@@ -76,6 +76,7 @@ export const SelectCapacityAndNodes: React.FC<SelectCapacityAndNodesProps> = ({
                 dispatch({ type: ActionType.SET_STORAGE_CLASS, payload: sc })
               }
               noSelection
+              selectedKey={getName(storageClass)}
```

Comment:
@afreen23 I checked the storageClassName is not set when the sc is created from step2. Looks like we are not setting it properly. I'll fix it with another PR setting the storageClassName properly.

Reply:
It is being set, otherwise localVolumeSet wont get created and you can never move to step 3
https://github.com/openshift/console/blob/eac92a317220e8bbcbe480a72dd8966dc0a678b5/frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/local-volume-set-inner.tsx#L76